www/include/configuration/configObject/hostgroup/listHostGroup.php

```diff
@@ -49,12 +49,20 @@
 /*
  * Search
  */
-$SearchTool = null;
-$search = "";
-if (isset($_POST['searchHg']) && $_POST['searchHg']) {
+$searchFilterQuery = null;
+$mainQueryParameters = [];
+$search = '';
+
+if (isset($_POST['searchHg'])) {
```

Comment:
missing the second condition of if

Reply:
If the condition is like this `isset($_POST['searchHg']) && $_POST['searchHg']` there would be no way to clear the search input, because if you submit an empty string it would get the old value and it will always do search.

Reply:
OK